src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/FunctionalQueueConfig.java

```diff
@@ -16,7 +16,7 @@
     @Value("${queue.envelopes.write-connection-string}")
     private String queueWriteConnectionString;
 
-    @Bean
+    @Bean({"envelopes", "processed-envelopes"})
```

Comment:
is the connection string to both queues the same??

Reply:
mistake - neither of them should be injected, actually